CONTRIBUTING.md

```diff
@@ -44,14 +44,19 @@ ___Do no setup for this requirement__ unless prompted to do so, and then follow
 - Install Homebrew if needed: https://brew.sh
 - Install python `brew install python`
 - Install [Poetry](https://python-poetry.org/docs/)
+- Install Docker
+  - This varies by each individual contributor's environment.
+  - Docker Desktop is the simplest
```

Comment:
We can't use docker desktop. It's a whole thing.  That's why we tell everyone to use Colima.

https://slalom.slack.com/archives/CG340903X/p1673907168152249

Reply:
Within Slalom, that's true, but for contributors outside of Slalom, that does not hold true. So I intentionally left it kind of vague